build.sbt

```diff
@@ -10,8 +10,8 @@ def previousVersion = "0.7.0"
 def scala213 = "2.13.2"
 def scala212 = "2.12.11"
 def scala211 = "2.11.12"
-def scala3Stable = "3.0.0-M2"
-def scala3Previous = List("3.0.0-M1", "0.27.0-RC1")
+def scala3Stable = "3.0.0-M3"
+def scala3Previous = List("3.0.0-M2", "3.0.0-M1")
```

Comment:
The new ScalaCheck release dropped support for 3.0.0-M1 so we'll need to drop here too (or at least that's the simplest option)

Reply:
I agree. Unless we have a very good reason, keeping only the two latest versions of Scala 3 around should be enough for smooth downstream support

Reply:
Removed it and left `3.0.0-M2` and `3.0.0-M3` only.